pontoon/base/migrations/0150_auto_20200103_0313.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-01-03 03:13
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0149_comment'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='comment',
+            name='translation',
+            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='comments', to='base.Translation'),
```

Comment:
Same here, not sure where it's coming from. Also, it would be better to just update the previous migration, that adds the comment table, rather than creating several migrations. It might be a bit more tricky for you, but for everyone else it will make a lot more sense to have only one (as we're going to squash all commits before merging).